app/models/concerns/phone_number_validator.rb

```diff
@@ -6,7 +6,7 @@ def validate_each(record, attribute, value)
     return if value.blank?
 
     phone = Phonelib.parse(value, 'US')
-    return unless phone.national(false).nil? || phone.national(false).length != 10
+    return unless phone.e164.blank? || phone.e164.sub(/^\+1/, '').length != 10
```

Comment:
I think there's still a case this isn't catching that it should: if a number has a country code _and_ it starts with 1, e.g. ` +1 (123) 456-7890`. The thinking being that if we don't allow `(123) 456-7890`, then shouldn't allow this either.

I think we'll get the desired effect if we check for one or more 1s at the start of the string:
```suggestion
    return unless phone.e164.blank? || phone.e164.sub(/^\+1+/, '').length != 10
```

I think this works fine for Import and API, but it would take us out of sync with the front end validation in the UI. If you make this change and then input `123-456-7890` in the Monitoree Contact Information page, you pass front end validation and fail back end validation, which results in this ugly situation:
![image](https://user-images.githubusercontent.com/8235974/106369298-4011a900-631e-11eb-9ceb-c70b6da1e405.png)

I don't want to get too out of scope here, but I think it would be a pretty simple modification to the front end validation here: https://github.com/SaraAlert/SaraAlert/blob/d8ef25b3eaef7308a971398a281546b0002b7120/app/javascript/components/enrollment/steps/Contact.js#L486

And then the same function is defined again for CloseContact:
https://github.com/SaraAlert/SaraAlert/blob/d8ef25b3eaef7308a971398a281546b0002b7120/app/javascript/components/close_contact/CloseContact.js#L239

Reply:
This makes sense to me. The one question I have is how far we want to go on the front end, because on the one hand since phone numbers can't start with `1`, I can't imagine we would really be hurting anybody by adding that constraint. On the other hand, it most people probably aren't aware of the constraint that phone numbers cannot start with `1` in the US (I know I wasn't).

So how would we feel about something like this:
![image](https://user-images.githubusercontent.com/52747882/106464001-1368a900-6466-11eb-805b-526f580f5669.png)
giving a specific error message for this case. And maybe this is something we then need to run by the PO team.

Reply:
For comparison, the normal error message just says "Please enter a valid Phone Number", which I think could be confusing for someone entering "123-456-7890".

Reply:
I think you're probably right that there should be an error message. With that specific message I would be slightly worried it could cause confusion since technically a phone number can start with 1 if you're including the country code. And perhaps the most likely scenario in which someone sees this message is if they tried to enter the number with the country code, not realizing the input box doesn't actually allow it. I would reword it, perhaps to say "10-digit Phone Number cannot start with 1", or maybe "Area code cannot start with 1". Would probably be a good idea to post in the UI slack channel.

Reply:
Ok, made this consistent in https://github.com/SaraAlert/SaraAlert/pull/582/commits/4a99df70839cbf69ca40a596d43d8c1b4436d940. Now the API/Import will not accept a phone number like `+1 123 456 7890`, and the UI will now allow a phone number to start with a `1`.

Reply:
LGTM